libs/modules/tasks/application/src/lib/tasks-service-factory.ts

```
@@ -0,0 +1,20 @@
+import { TasksRepository } from '@tasks/domain';
+
+import { TasksService } from './tasks.service';
+
+export class TasksServiceFactory {
+  private static instance: TasksService | null = null;
+
+  static initialize(repository: TasksRepository): void {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)